gnd/src/main/java/com/google/android/gnd/repository/FeatureRepository.java

```diff
@@ -127,21 +127,33 @@ private FeatureMutation fromFeature(Feature feature, Type type) {
   }
 
   // TODO(#80): Update UI to provide FeatureMutations instead of Features here.
-  public Completable createFeature(Feature feature) {
-    return applyAndEnqueue(fromFeature(feature, Type.CREATE));
+  public Flowable<Loadable<Feature>> createFeature(Feature feature) {
```

Comment:
I've experimented with pushing the Flowable<Loadable<>> down in the stack like this in the past, and while it simplifies usage, it also complicates the semantics of the internal API; these methods no longer simply do work and then either complete or return an error, but rather they stream status. This is slightly harder for clients of these methods to understand, and is really only introduced here because of how the UI streams emit state. It also is questionable whether this simplifies the code, since the transformation to a Flowable still has to happen somewhere.

Can we wrap the state higher up in the stack instead, as close to the subscription as possible?

Also note that `Loadable` is modeled around the action of something being loaded from the database, not creating, updating, and deleting.